share/spice/cargo/content.handlebars

```diff
@@ -0,0 +1,3 @@
+<pre>[dependencies]
```

Comment:
The dependencies are automatically resolved, I don't think there's a need to list each one. The code in the pre block is meant to be in the project build file